src/linux/backend-v4l2.cpp

```diff
@@ -1130,147 +1130,155 @@ namespace librealsense
                             {
                                 LOG_DEBUG_V4L("Dequeued empty buf for fd " << std::dec << _fd);
                             }
+
                             LOG_DEBUG_V4L("Dequeued buf " << std::dec << buf.index << " for fd " << _fd << " seq " << buf.sequence);
 
                             auto buffer = _buffers[buf.index];
                             buf_mgr.handle_buffer(e_video_buf, _fd, buf, buffer);
 
-                            if (_is_started)
+                            if (buf.flags & V4L2_BUF_FLAG_ERROR)
+                            {
+                                LOG_DEBUG_V4L("VIDEO - VIDIOC_DQBUF flags with flag = " << std::hex << buf.flags <<", index = " << std::dec << buf.index << " for fd " << _fd << " seq " << buf.sequence);
```

Comment:
Do we want an error log here? or the logs concept is only debug on this file?
Ether way I think we can add an indication in the message, something like:
`Failed on handle_buffer with flag...`
What do you think?

Reply:
The LOG_DEBUG_V4L is indeed only to debug the v4l2 backend, and it is usually not enabled.
I agree that some error text would be  better there, I ll correct that.